src/releng_docs/deploying.rst

```diff
@@ -1,146 +1,244 @@
+.. _`Firefox release week`: https://wiki.mozilla.org/RapidRelease/Calendar
 
-Secrets are retrived from `taskcluster secrets services`_ for each branch
-separatly.
 
-- staging => `repo:github.com/mozilla-releng/services:branch:staging`_
-- production => `repo:github.com/mozilla-releng/services:branch:production`_
+Release Protocol
+----------------
 
+This protocal is followed by the :ref:`release manager <release-managers>` of ``mozilla-releng/services``.
 
-URL naming scheme
------------------
+#. If not already there, create a PR from ``master`` to ``production`` branch.
+   (`Example of release PR`_).
+
+   Please make sure that release notes are collected in description of the PR.
+   Encurage developers adding release notes as their PR gets merged into
+   ``master``.
+
+#. A day prior to the scheduled release send an email to all :ref:`service
+   owners <service-owners>`
+
+   This email should announce and remind everybody that release is going to
+   happen to avoid surprises.
+   
+   Template for email::
+
+       SUBJECT:
+
+           mozilla-releng/services release v<VERSION> is going to
+           happen on <RELEASE_DATETIME>
+
+       BODY:
+
+           Hi,
+
+           Next planned mozilla-releng/services relase is going to
+           happen tomorrow. We encurage everybody that contributed
+           to this release to join "#shipit" channel where release
+           is going to be coordinated. 
+
+               YYYY-MM-DD MM:HH CET
+               YYYY-MM-DD MM:HH PST
+
+           Please follow the link bellow for more details.
+
+               <LINK-TO-RELEASE-PR>
+
+           For any question please contact
+
+               <CURRENT-RELEASE-MANAGER>
+
+          
+#. Release starts by :ref:`release manager <release-managers>` loging all the
+   steps into ``#shipit`` channel and coordinating with others.
+
+#. Push to ``staging`` branch and do (if needed) some manual checks.
 
-URL naming scheme of deployed services is as following:
+   .. code-block:: shell
 
-- ``src/releng_frontend``
+        $ git clone git@github.com/mozilla-releng/services.git
+        $ cd services
+        $ git checkout -b staging origin/staging
+        $ git push origin staging -f 
 
-  - staging => https://staging.mozilla-releng.net
-  - production => https://mozilla-releng.net
+#. Push to ``production`` branch and do (if needed) some manual checks.
+   
+   Create a merge commig (Example of merge commit) of master branch and tag it
 
-- ``src/releng_SERVICE``
+   .. code-block:: shell
 
-  - staging => https://SERVICE.mozilla-releng.net
-  - production => https://SERVICE.staging.mozilla-releng.net
+        $ git clone git@github.com/mozilla-releng/services.git
+        $ cd services
+        $ git checkout -b productiona origin/production
+        $ git merge master -m "Release: v13"
+        $ git push origin production
+        $ git tag v13
+        $ git push origin v13
```

Comment:
It might be better to stub out versions here instead of including existing ones. People may copy/paste these, and forget to update the number.